cardano-cli/src/Cardano/CLI/EraBased/Options/Common.hs

```diff
@@ -8,6 +8,7 @@
 {- HLINT ignore "Move brackets to avoid $" -}
 {- HLINT ignore "Use <$>" -}
 {- HLINT ignore "Move brackets to avoid $" -}
+{- HLINT ignore "Use section" -}
```

Comment:
what's wrong with sections?

Reply:
It yields this suggestion:

```
cardano-cli/src/Cardano/CLI/EraBased/Options/Common.hs:538:26-36: Suggestion: Use section
Found:
  ((<>) ": ")
Perhaps:
  (": " <>)
```

which I personally don't find a really good change. But no strong opinion. I can leave it back if you prefer.

Reply:
They're fine imo. My problem is with per-module level hlint rules. I think we should agree together on a set of them, put them into the top level config and enforce those rules everywhere.